src/include/uapi/ipc/dai-intel.h

```diff
@@ -112,6 +112,7 @@ struct sof_ipc_dai_ssp_params {
 struct sof_ipc_dai_hda_params {
 	struct sof_ipc_hdr hdr;
 	uint32_t link_dma_ch;
+	uint32_t reserved[4];
```

Comment:
Not sure I understand - how is this related to the fix? Is this the reason to change the ABI minor number? If the answer to both questions is "yes," I think it's better to remove them from this patch

Reply:
@lyakh this is a pretend ABI update to track this change. With this update to link DMA assignment, the kernel/FW are officially backwards incompatible

Reply:
For incompatibility we have to increment the major version, don't we? Still you don't need to add those reserved fields, if they're unrelated.

Reply:
@lyakh that was my original change but this is what has been agreed with maintainers. A major ABI change usually requires big changes to the IPC structs. I was told functional incompatibility cannot be classified a major ABI change. So, we advertise full HDA support only 3.8 onwards.

Reply:
@ranj063 this and the ABI bump are not needed.